### PR TITLE
tools/dm: update some sentences found inaccurate or unclear

### PR DESCRIPTION
**Pull request by dcalvin**
Updated several sentences that I found inaccurate or unclear while translating the file into Chinese

### CODE REVIEW

File: tools/dm/cluster-operations.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ For the binlog during incremental data import, DM uses the downstream database t
 
     - When DM-worker is restarted before or after synchronizing sharding DDL statements, it checks the checkpoint information and you can use the `start-task` command to recover the data synchronization task automatically.
 
-    - When DM-worker is restarted during the process of synchronizing sharding DDL statements, the issue might occur that the DM-worker owner has executed the DDL statement and successfully changed the downstream database table schema, while other DM-worker instances are restarted but fail to skip the DDL statement and update the checkpoints.
+    - When DM-worker is restarted during the process of synchronizing sharding DDL statements, the issue might occur that the owner DM-worker has executed the DDL statement and successfully changed the downstream database table schema, while other DM-worker instances are restarted but fail to skip the DDL statement and update the checkpoints.
```
Comment (csuzhangxc) [inline tools/dm/cluster-operations.md]:
```suggestion
    - When DM-worker is restarted during the process of synchronizing sharding DDL statements, the issue might occur that the owner (one of DM-worker instances) has executed the DDL statement and successfully changed the downstream database table schema, while other DM-worker instances are restarted but fail to skip the DDL statement and update the checkpoints.
```

File: tools/dm/cluster-operations.md [hunk 2]
```diff
@@ -355,7 +355,7 @@ Assuming that the `172.16.10.72` machine needs to be maintained or this machine
 
 3. Edit the `inventory.ini` file and add the new DM-worker instance.
 
-    Edit the `inventory.ini` file, comment or delete the line where the `dm_worker1` instance `172.16.10.72` that you want to replace exists, and add the `172.16.10.75` information of the new `dm_worker1` instance.
+    Edit the `inventory.ini` file, comment or delete the line where the original `dm_worker1` instance that you want to replace exists, and add the information for the new `dm_worker1` instance.
```
Comment (csuzhangxc) [inline tools/dm/cluster-operations.md]: how about
```suggestion
    Edit the `inventory.ini` file, comment or delete the line where the original `dm_worker1` instance (`172.16.10.72`) that you want to replace exists, and add the information for the new `dm_worker1` instance (`172.16.10.75`).
```

**Comment by csuzhangxc** [comment]
LGTM

File: tools/dm/cluster-operations.md [hunk 3]
```diff
@@ -395,7 +395,7 @@ This section describes how to switch between master and slave instances using dm
 
 ### Upstream master-slave switch behind the virtual IP
 
-1. Use `query-status` to make sure that relay catches up with the master instance before the switch (`relayCatchUpMaster`).
+1. Use `query-status` to make sure that the relay unit has caught up with binlog status of the master instance before the switch (`relayCatchUpMaster`).
```
Comment (lilin90) [inline tools/dm/cluster-operations.md]:
```suggestion
1. Use `query-status` to make sure that the relay unit has caught up with the binlog status of the master instance before the switch (`relayCatchUpMaster`).
```

File: tools/dm/cluster-operations.md [hunk 4]
```diff
@@ -405,8 +405,8 @@ This section describes how to switch between master and slave instances using dm
 
 ### Master-slave switch after changing IP
 
-1. Use `query-status` to make sure that relay catches up with the master instance before the switch (`relayCatchUpMaster`).
+1. Use `query-status` to make sure that the relay unit has caught up with binlog status of the master instance before the switch (`relayCatchUpMaster`).
```
Comment (lilin90) [inline tools/dm/cluster-operations.md]:
```suggestion
1. Use `query-status` to make sure that the relay unit has caught up with the binlog status of the master instance before the switch (`relayCatchUpMaster`).
```

**Pull request merged**